### ISSUE TITLE
actions-executing plug-in is not working as expected with nx monorepo !

### ISSUE THREAD

**Issue by HD60Hz** [ngxs-labs/actions-executing#15]
We recently moved our Angular project into a monorepo (nx). we noticed that the action execution plugin no longer works!
are there any special configurations when it comes to using this kind of architectures, especially with shared libraries etc. ?

**Comment by joaqcid** [comment]
hi @HD60Hz i use this plugin myself with Nx without any issues, can you be more specific what issue you are having?

btw, sorry for late response

**Comment by joaqcid** [comment]
closing the issue, if you still are having problem please post more details and if possible a stackblitz with the error repro.

thanks @HD60Hz

**Comment by MahmoudTarekAli** [comment]
Facing the Same issue here after i moved my angular project to nx workspace the action execution plugin no longer works! it always returns false and doesn't detect the changes which should be false => true => false
 and hasActionsExecuting return undefined
   
   ```
   this.store
      .select((state) => state['hasActionsExecuting'])
      .subscribe((data) => {
        console.log('Actions Executing:', data);
      });

**Comment by joaqcid** [comment]
hi @MahmoudTarekAli 

that is not how you are supposed to use the actionExecuting plugin

Please refer to the documentation

But generally speaking you would 
`this.store.select(hasActionsExecuting([ACTION]))`

**Comment by MahmoudTarekAli** [comment]
Okay  but I am also using the @select decorator but facing the same issue, so is this not working anymore ?

 @Select(hasActionsExecuting([GetFeaturedArenas]))
  getFeaturedArenasIsExecuting$: Observable<ActionsExecuting>;

**Comment by MahmoudTarekAli** [comment]
and after using this 
this.store.select(hasActionsExecuting([ACTION]))

it also keeps return only false not   false => true => false

**Comment by joaqcid** [comment]
@MahmoudTarekAli yes, the plugin works with NX, my project uses nx version 17 and works ok.

what versions of nx, ngxs and actions-executing plugin are you using?

would you be able to create a repo with a reproducible error?

**Comment by joaqcid** [comment]
@MahmoudTarekAli @HD60Hz NX shouldnt change anything about how this plugin works, the only thing i would make sure is what angular version you are using and making sure is aligned with the ivy or non-ivy version of the plugin

**Comment by joaqcid** [comment]
@MahmoudTarekAli you still have issue with this?
